### PR TITLE
MSYS-832 : Added Max_body_size configurable

### PR DESCRIPTION
**Pull request by piyushawasthi**
Signed-off-by: piyushawasthi <piyush.awasthi@msystechnologies.com>

### CODE REVIEW

File: apps/pushy/src/pushy_config_resource.erl [hunk 1]
```diff
@@ -39,6 +39,7 @@
 -include("pushy_wm.hrl").
 
 -define(DEFAULT_CONFIG_LIFETIME, 3600).
+-define(MAX_BODY_SIZE, 65536).
```
Comment (markan) [inline apps/pushy/src/pushy_config_resource.erl]: Note that this default is different from the default in push client for a reason. In the server we measure the size of the whole packet, while in the client we measure the size of the payload. We're going to have to reconcile the two or else we're going to introduce a bug.

Comment (markan) [reply apps/pushy/src/pushy_config_resource.erl]: For reference that check happens on the erlang side here: https://github.com/chef/pushy_common/blob/master/src/pushy_messaging.erl#L114

**Comment by rhass** [comment]
@piyushawasthi Where are we with this?

**Comment by piyushawasthi** [comment]
@rhass : i have done with that waiting for @markan review.

**Pull request merged**